plugins/poll/assets/javascripts/discourse/templates/modals/poll-ui-builder.hbs

```diff
@@ -0,0 +1,58 @@
+<div class="modal-body poll-ui-builder">
+  <form class="poll-ui-builder-form form-horizontal">
+    <div class="input-group">
+      <label>{{i18n 'poll.ui_builder.poll_name.label'}}</label>
+      {{input name="poll-name" value=pollName placeholder=pollNamePlaceholder}}
+    </div>
+
+    <div class="input-group">
+      <label>{{i18n 'poll.ui_builder.poll_type.label'}}</label>
+      {{combo-box content=pollTypes
+                  value=pollType
+                  valueAttribute="value"
+                  none="poll.ui_builder.poll_type.regular"}}
+
+      {{#if showMinMax}}
+        <label>{{i18n 'poll.ui_builder.poll_config.min'}}</label>
+        {{combo-box content=pollMinOptions
+                    value=pollMin
+                    valueAttribute="value"
+                    class="poll-options-min"}}
+
+        <label>{{i18n 'poll.ui_builder.poll_config.max'}}</label>
+        {{combo-box content=pollMaxOptions
+                    value=pollMax
+                    valueAttribute="value"
+                    class="poll-options-max"}}
+
+        {{#if isNumber}}
+          <label>{{i18n 'poll.ui_builder.poll_config.step'}}</label>
+          {{combo-box content=pollStepOptions
+                      value=pollStep
+                      valueAttribute="value"
+                      class="poll-options-step"}}
+        {{/if}}
+      {{/if}}
+    </div>
+
+    <div class="input-group">
+      <label for="poll-public">
+        {{input type='checkbox' checked=publicPoll}}
+        {{i18n "poll.ui_builder.poll_public.label"}}
+      </label>
+    </div>
+
+    {{#unless isNumber}}
+      <div class="input-group">
+        <label>{{i18n 'poll.ui_builder.poll_options.label'}}</label>
+        {{input-tip validation=minNumOfOptionsValidation}}
+        {{d-editor value=pollOptions}}
+      </div>
+    {{/unless}}
+  </form>
+</div>
+
+<div class="modal-footer">
+  {{d-button action="insertPoll" class='btn btn-primary' label='poll.ui_builder.insert' disabled=disableInsert}}
```

Comment:
`d-button` shouldn't need the `btn` class. It has it by default. `primary` is still required though.